cypress/e2e/downloads.cy.ts

```diff
@@ -0,0 +1,369 @@
+import {PLAYER_CONFIG} from '../e2e/mock-data';
+
+interface KalturaPlayer {
+  setup: (config: any) => Player;
+}
+
+interface Player {
+  setMedia: (arg0: {session: {ks: string}; sources: {id: string; progressive: {mimetype: string; url: string}[]}}) => void;
+  isLive: () => boolean;
+  getVideoElement: () => {mediaKeys: any};
+  ready: () => any;
+}
+
+interface Window {
+  KalturaPlayer: KalturaPlayer;
+}
+
+let player: Player | null;
+let requestCount = 0;
+
+const loadPlayer = (playerConfig?: any): Promise<void> => {
+  return new Promise(resolve => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    cy.visit('index.html').then((win: Window) => {
+      if (playerConfig) {
```

Comment:
if you have default value for playerConfig (empty object) you can remove if/else condition
line `player = win.KalturaPlayer.setup({...PLAYER_CONFIG, ...playerConfig});` will works in any cases

Reply:
fixed